src/Smalot/PdfParser/Font.php

```diff
@@ -86,7 +86,12 @@ public function getDetails($deep = true)
 
         $details['Name']     = $this->getName();
         $details['Type']     = $this->getType();
-        $details['Encoding'] = ($this->has('Encoding') ? (string)$this->get('Encoding') : 'Ansi');
+
+        try {
+            $details['Encoding'] = ($this->has('Encoding') ? (string) $this->get('Encoding') : 'Ansi');
+        } catch (\Exception $e) {
+            $details['Encoding'] = 'Ansi';
```

Comment:
I dislike the try-catch approach, because it will hide any exception thrown. One will never know if an exception was thrown was relevant or not.